tests/project/_common/test_movement.py

```diff
@@ -0,0 +1,437 @@
+        ).resource_id
+        r = client.delete(f"/api/versioned/v1/datasets/{id_3}?cascade=false")
+        responses.append(r)
+    except Exception as e:
+        print(f"Error: {str(e)}")
+
+    try:
+        id_4 = client.datasets.by_name(unified_dataset_name + "_record_comments").resource_id
+        r = client.delete(f"/api/versioned/v1/datasets/{id_4}?cascade=false")
+        responses.append(r)
+    except Exception as e:
+        print(f"Error: {str(e)}")
+
+    try:
+        id_5 = client.datasets.by_name(
+            unified_dataset_name + "_classification_histogram_boundaries"
+        ).resource_id
+        r = client.delete(f"/api/versioned/v1/datasets/{id_5}?cascade=false")
+        responses.append(r)
+    except Exception as e:
+        print(f"Error: {str(e)}")
+
+    try:
+        id_6 = client.datasets.by_name(unified_dataset_name + "_classification_model").resource_id
+        r = client.delete(f"/api/versioned/v1/datasets/{id_6}?cascade=false")
+        responses.append(r)
+    except Exception as e:
+        print(f"Error: {str(e)}")
+
+    return responses
+
+
+@mock_api()
+def test_export_errors():
+    client = utils.client.create(**CONFIG["toolbox_test_instance"])
+    project = client.projects.by_resource_id(CONFIG["projects"]["minimal_categorization"])
+
+    # test incorrect artifact name
+    with pytest.raises(ValueError):
+        categorization.export_artifacts(
+            project=project,
+            artifact_directory_path="/home/ubuntu/tamr/projectExports",
+            exclude_artifacts=[
+                catfacts.CATEGORIZATION_VERIFIED_LABELS,
+                catfacts.CATEGORIZATION_TAXONOMIES,
+                "INCORRECT_ARTIFACT_NAME",
+            ],
+            asynchronous=False,
+        )
+
+    # test incorrect artifact directory path
+    with pytest.raises(ValueError):
+        categorization.export_artifacts(
+            project=project,
+            artifact_directory_path="/an/incorrect/path",
+            exclude_artifacts=[
+                catfacts.CATEGORIZATION_VERIFIED_LABELS,
+                catfacts.CATEGORIZATION_TAXONOMIES,
+            ],
+            asynchronous=False,
+        )
+
+
+@mock_api()
+def test_import_new_errors():
+    client = utils.client.create(**CONFIG["toolbox_test_instance"])
+    project = client.projects.by_resource_id(CONFIG["projects"]["minimal_categorization"])
+
+    # export a project
+    op = categorization.export_artifacts(
+        project=project,
+        artifact_directory_path="/home/ubuntu/tamr/projectExports",
+        exclude_artifacts=None,
+        asynchronous=False,
+    )
+
+    assert op.succeeded()
+
+    # get artifact path
+    description = op.description
+    artifact_path = str(description).split(" ")[-1]
+
+    assert artifact_path[0] == "/"
+    assert artifact_path[-4:] == ".zip"
+
+    # import new project if it doesn't exist already
+    new_project_name = "new_categorization"
```

Comment:
Should you add a timestamp to your new_project_name to make it trully unique?

Reply:
I tried this and it works for live tests, but it becomes an issue when attempting to use the mock_api offline because every new test generates new timestamps, but the saved responses used by mock_api refer to past time-stamps, and the offline tests fail. 

For now I'm keeping the tests with definite pre-determined project names, without timestamp.